active_learning/views.py

```diff
@@ -16,35 +16,59 @@
 
 @xframe_options_exempt
 def index(request):
+    '''
+    Renders the list of articles on the index.html page
+    '''
+
     articles = Article.objects.all().order_by("class_label_id")
     context = {'articles': articles, 'count': len(articles)}
     return render(request, 'active_learning/index.html', context)
 
 
 @xframe_options_exempt
 def detail(request, article_id):
+    '''
+    Renders an articles details on the details.html page
+    '''
+
     article = get_object_or_404(Article, pk=article_id)
     return render(request, 'active_learning/detail.html', {'article': article})
 
 
 def iframe(request):
+    '''
+    Renders the iframe in the popup iframe.html page
+    '''
+
     presence_channel_name = format_pusher_channel_name(environ['PRESENCE_CHANNEL_NAME'])
     PUSHER_KEY = environ['PUSHER_KEY']
     return render(request, 'active_learning/iframe.html', {'presence_channel_name': presence_channel_name, 'PUSHER_KEY': PUSHER_KEY})
 
 
 def learn(request):
+    '''
+    After retrieving an oracle's response, learn from the enw data
```

Comment:
starts the active learning process which requires another tab to be loaded at the active_learning root (iframe) page for the oracle to label articles